docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+  - incremental = appends a structured -nnn suffix to the original tag, where -nnn increments from 1 to 999. The highest existing -nnn suffix will always be the one incremented for subsequent patches
+  - floating = creates a single mutable tag formatted as "-patched" that is appended to the original tag and always points to the latest patched image
+
+    We recommend using incremental tags as a more secure method of updating patches for deployments. Incremental tags are immutable, which aligns with cloud native best practices (easy rollbacks).
+
+    As for floating tags, they can be used conveniently for builds at build time, since the consistent tag always points to the latest patch and allows teams to avoid the constant manual switch to new patched tags.
+  
+    Here's an example:
+  
+    ![PatchingTimelineExample](./media/patching_timeline_example1.png)
+
+- "repositories" is an array that consists of all objects that detail the specific repository and tags
+    - "repository" refers to repository name
+    - "tags" is an array of tags separated by commas. The wildcard "*" can be used to signify all tags within that repository
+    - "enabled" is a Boolean value of true or false determining if the specified repo is on or off
+
+The following details an example configuration for a customer who wants to patch all tags(use the * symbol) within the repository “python”, and to patch specifically the “jammy-20240111” and “jammy-20240125” tags in the repository “ubuntu”. 
+
+JSON example:
+```json
+{
+"version": "v1",
+"tag-convention" : "incremental",
+"repositories": [{
+        "repository": "python",
+        "tags": ["*"],
+        "enabled": true
+    },
+    {
+        "repository": "ubuntu",
+        "tags": ["jammy-20240111", "jammy-20240125"],
+        "enabled": true, 
+    }]
+}
+```
+4. After creating your configuration file, it is recommended to execute a dry run to verify the intended artifacts are selected by the JSON criteria. The dry run requires a parameter called schedule, which specifies how often your continuous patching cycle will run. The schedule flag is measured in days, with a minimum value of 1 day, and a maximum value of 30 days. 
+
+Command Schema:
+```sh
+az acr supply-chain workflow create -r <registryname> -g <resourcegroupname> -t continuouspatchv1 --config <JSONfilepath> --schedule <number of days> --dry-run 
+```
+Example Command 
+```sh
+az acr supply-chain workflow create -r myRegistry -g myResourceGroup -t continuouspatchv1 -–config ./continuouspatching.json --schedule 1d –-dry-run   
+```
+Help command to see all required/optional flags
+```sh
+az acr supply-chain workflow create --help
+```
+ 
+This command will output all specified artifacts by the JSON file configuration. Customers can verify that the right artifacts are selected. With the sample ubuntu configuration above, the following results should be displayed as output. 
+```sh
+Ubuntu: jammy-20240111
+Ubuntu: jammy-20249125
+```
```

Comment:
"This command will output all specified artifacts ..."

The command directly above this is the help command. Please reorder so that the help command comes after. Otherwise, the customer will be confused if they look at this section:

![image](https://github.com/user-attachments/assets/50382327-f46c-4012-b501-d07e67452eca)